evalml/objectives/fraud_cost.py

```diff
@@ -56,20 +61,19 @@ def objective_function(self, y_true, y_predicted, X):
 
         Arguments:
             y_predicted (pd.Series): Predicted fraud labels
-             y_true (pd.Series): True fraud labels
-            X (pd.DataFrame): pd.Dataframe with transaction amounts
+            y_true (pd.Series): True fraud labels
+            X (ww.DataTable, pd.DataFrame): Data with transaction amounts
 
         Returns:
             float: Amount lost to fraud per transaction
         """
+        if isinstance(X, ww.DataTable):
+            X = _convert_woodwork_types_wrapper(X.to_dataframe())
         if not isinstance(X, pd.DataFrame):
             X = pd.DataFrame(X)
-
-        if not isinstance(y_predicted, pd.Series):
-            y_predicted = pd.Series(y_predicted)
-
-        if not isinstance(y_true, pd.Series):
-            y_true = pd.Series(y_true)
+        y_true = self._standardize_input_type(y_true)
```

Comment:
Ah, it wasn't previously because `_standardize_input_type` only handled 1D inputs. Updated so this is no longer the case, and centralizing our standardizing logic in `_standardize_input_type` :)